noggin/form/edit_user.py

```diff
@@ -43,7 +45,7 @@ class UserSettingsProfileForm(FlaskForm):
         ],
     )
 
-    ircnick = StringField(_('IRC Nickname'), validators=[Optional()])
+    ircnick = CSVListField(_('IRC Nicknames'), validators=[Optional()])
```

Comment:
Maybe add a help text here explaining that this should be comma-separated, otherwise it's not very obvious I think.

Reply:
in the UI here we are using selectize, so the user shouldnt ever have to comma separate the values